crates/holochain/src/sweettest/sweet_conductor.rs

```diff
@@ -988,6 +989,19 @@ impl SweetConductor {
             Ok(all_integrated)
         })
     }
+
+    /// Manually trigger scheduled fn dispatch
+    pub async fn dispatch_scheduled_fns(&self, now: Timestamp) {
+        self.raw_handle().dispatch_scheduled_fns(now).await;
+    }
+
+    /// Manually start the scheduler
+    pub async fn start_scheduler(
```

Comment:
These two methods are needed for integration tests of the `schedule` host fn. They are private in `Conductor` and since the tests were previously within the crate as unit-tests this was not a problem. So the solution here is to make them available publicly via `SweetConductor`.

Reply:
They are already `pub(crate)` in `Conductor`, but `pub(crate)` is not accessible to the crate's integration test. Thats why.